README.rst

```diff
@@ -23,20 +23,21 @@ Description
 
 This is a tool for uploading certain kinds of files to CGAP.
 
-Initial support is for "metadata bundles", which are Excel files
-(such as .xls or .xlsx files)
-that are accompanied by other files (such as ``.fastq.gz`` files).
+Current support is for "metadata bundles" and "gene lists".
+"Metadata bundles" are Excel files (.xlsx) accompanied by other files 
+(such as ``.fastq.gz`` files). 
+"Gene lists" are either Excel files (.xlsx) or plain text (.txt) files.
 
 
 About Metadata Bundles
 ======================
+"Metadata bundles" are Excel files (.xlsx) accompanied by other files 
+(such as ``.fastq.gz`` files). 
 
-.. note::
-
-   The format of the Excel files that are used as
-   "metadata bundles" is not yet documented.
-   For now you should begin by obtaining a template file from
-   your contact on the CGAP Team and then customize that as appropriate.
+Note: The format of the Excel files that are used as
+"metadata bundles" is not yet documented.
+For now you should begin by obtaining a template file from
+your contact on the CGAP Team and then customize that as appropriate.
```

Comment:
Indeed, when I view the current repo page (running Firefox 87.0) I can't see the appropriate note box (see the attached screenshot), nor when I ran it through a different renderer (not PyCharm's) was I able to see the note box. 

I don't know why it's not appearing, but I changed it because I thought it had possibly been an oversight. Does it appear correctly for you on the current repo page? 


<img width="908" alt="Screen Shot 2021-04-15 at 2 36 12 PM" src="https://user-images.githubusercontent.com/58236592/114921385-81333880-9df8-11eb-8453-904c8653dce8.png">